_pages/en_US/homebrew-launcher-(super-skaterhax).txt

```diff
@@ -0,0 +1,17 @@
+---
+title: "Homebrew Launcher (super-skaterhax)"
+---
+
+{% include_relative include/super-skaterhax.txt %}
+
+___
+
+### Next steps: Choose an exploit
+
+If you saw the system update screen in the previous section, Safe Mode is working on your device.
```

Comment:
in what section on this page do we do a safe mode check?

also, can we make nimhax inline troubleshooting too, this branching makes no sense. this isn't the wiki, we pick a route first and only if it doesn't work we do something else, whereas people reading up on older exploits on the wiki probably know at least the baseline of what to expect already

Reply:
the branching here makes sense, but it's a bit of a maze. Not much we can do unless we don't use this method